python/tests/adjoint_solver.py

```diff
@@ -0,0 +1,150 @@
+import meep as mp
+import meep.adjoint as mpa
```

Comment:
Just `import adjoint as mpa` to import from the local path.

Reply:
Done but unfortunately now this test can only be run as part of `make check`. Trying to run the test independently produces an error when loading the `adjoint` module.

Reply:
Do:
```py
try:
    import meep.adjoint as mpa
except:
    import adjoint as mpa
```